pkg/datapath/linux/probes/managed_neighbors.go

```diff
@@ -0,0 +1,104 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package probes
+
+import (
+	"errors"
+	"fmt"
+	"net"
+	"runtime"
+	"sync"
+
+	"github.com/vishvananda/netlink"
+	"github.com/vishvananda/netns"
+)
+
+var (
+	managedNeighborOnce   sync.Once
+	managedNeighborResult error
+)
+
+// HaveManagedNeighbors returns nil if the host supports managed neighbor entries (NTF_EXT_MANAGED).
+// Any errors encountered during the feature probe are returned verbatim.
```

Comment:
This is no longer true. Let's add the same note as on the other Have...() functions:
```suggestion
// On unexpected probe results this function will terminate with log.Fatal().
```